thingsboard_gateway/extensions/tcpserver/tcpserver_connector.py

```diff
@@ -0,0 +1,243 @@
+#     Copyright 2020. ThingsBoard
+#
+#     Licensed under the Apache License, Version 2.0 (the "License");
+#     you may not use this file except in compliance with the License.
+#     You may obtain a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#     Unless required by applicable law or agreed to in writing, software
+#     distributed under the License is distributed on an "AS IS" BASIS,
+#     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#     See the License for the specific language governing permissions and
+#     limitations under the License.
+
+
+import threading
+from thingsboard_gateway.tb_utility.tb_utility import TBUtility
+from socketserver import ThreadingTCPServer, BaseRequestHandler
+import binascii
+import time
+from thingsboard_gateway.connectors.connector import Connector, log
+from thingsboard_gateway.extensions.tcpserver.tcpserver_converter import BytesTcpUplinkConverter
+
+
+global TCPServer_gateway
+global TCPServer_devices
+RequestHandlerDict = {}
+
+
+class MyBaseRequestHandler(BaseRequestHandler):
+    def setup(self):
+        self.__devices = TCPServer_devices
+        print("self.client_address:", self.client_address)
```

Comment:
Please use the logging module to print the data(log.debug(...), log.info(...) etc.) instead of "print" function (It doesn't send the output into the log files).

Reply:
ok